CONTRIBUTORS

```diff
@@ -8,4 +8,6 @@ Google Inc.
  Andrey Konovalov
  David Drysdale
  Baozeng Ding
-Lorenzo Stoakes
+ Lorenzo Stoakes
```

Comment:
Revert this line. Lorenzo is an individual contributor, so he should not be listed under Google.
I agree while here are few entries, the structure is not obvious.